recipes-kernel/linux/linux-tinker-board.bb

```diff
@@ -0,0 +1,20 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+LINUX_VERSION = "4.4.103"
+
+SRC_URI = " \
+    git://github.com/TinkerBoard/debian_kernel.git;protocol=https;branch=release \
+    file://defconfig \
+    "
+
+SRCREV = "2.0.7"
+
+require recipes-kernel/linux/linux-yocto.inc
+
+PV = "${LINUX_VERSION}"
+
+S = "${WORKDIR}/git"
+
+#KCONFIG_MODE="--alldefconfig"
```

Comment:
If this is not needed, let's remove it altogether

Reply:
Actually, if you comment it out you can then also remove adding the `defconfig` in the layer.
And this would be the preferred approach in fact.
So let's comment out this line and remove the defconfig